Sources/MuxPlayerSwift/PublicAPI/Options/PlaybackOptions.swift

```diff
@@ -75,6 +75,89 @@ public enum SingleRenditionResolutionTier {
     case only2160p
 }
 
+/// Creates an instant clip of an asset without re-encoding.
+/// Available for all video on-demand assets.
+public struct InstantClipping: Equatable {
+
+    let noInstantClipping: Bool
+    var assetStartTimeInSeconds: UInt?
+    var assetEndTimeInSeconds: UInt?
+
+    /// Omits instant clipping when loading a playback ID.
+    public static let none = InstantClipping()
+
+    init() {
+        self.assetStartTimeInSeconds = nil
+        self.assetEndTimeInSeconds = nil
+        self.noInstantClipping = true
+    }
+
+    /// Streams a clip whose starting time are based on
+    /// the relative time of the asset.
+    ///
+    /// A few extra seconds of video may be included in the
+    /// clip. Example:
+    ///
+    /// ```swift
+    /// let instantClipping = InstantClipping(
+    ///     assetStartTimeInSeconds: 10,
+    ///     assetEndTimeInSeconds: 20
+    /// )
+    /// ```
+    ///
+    /// Will stream a clip that starts about 10 seconds
+    /// after your asset starts or a little earlier, and
+    /// finishes approximately 10 more seconds after that.
+    ///
+    /// Clips an asset based on its relative time. A few
+    /// extra seconds of video may be included in the clip.
+    ///
+    /// - Parameters:
+    ///   - assetStartTimeInSeconds: the starting time
+    ///   of the clip relative to the asset time
+    ///   - assetEndTimeInSeconds: the ending time of
+    ///   the clip relative to the asset time
+    public init(
+        assetStartTimeInSeconds: UInt,
```

Comment:
The asset relative times were `Double` but decided to use `UInt` because these clips can include a few extra seconds of video outside of the asset start and end time.

Decided to omit using [`CMTime`](https://developer.apple.com/documentation/coremedia/cmtime-u58?language=swift) that's commonly used to express playback time here for the same reason, since it is a high-precision timestamp.